.github/workflows/ci.yml

```diff
@@ -39,15 +46,15 @@ jobs:
           node-version: 20
           cache: yarn
 
-      # Build main arch and run tests.
+      # Build and run tests.
       - run: yarn install --frozen-lockfile
       - run: yarn test
 
-      # Build secondary arch.
-      - run: npx node-gyp rebuild --arch=${{ matrix.arch2 }}
-        if: ${{ matrix.arch2 != 'none' }}
-      - run: node scripts/build-sub-package.js ${{ matrix.arch2 }}
-        if: ${{ matrix.arch2 != 'none' }}
+      # Build arm64 for Windows.
+      - run: npx node-gyp rebuild --arch=${{ matrix.arch }}
+        if: ${{ matrix.name == 'win32' && matrix.arch != 'arm64' }}
+      - run: node scripts/build-sub-package.js ${{ matrix.arch }}
+        if: ${{ matrix.name == 'win32' && matrix.arch != 'arm64' }}
```

Comment:
Is `&& matrix.arch != 'arm64'` really needed here? We exclude win32 arm64 above.

Reply:
Good catch. It should be `matrix.arch == 'arm64'`

Reply:
Good catch! We should pass it explicitly